packages/vscode-tailwindcss/src/extension.ts

```diff
@@ -609,7 +609,7 @@ export async function activate(context: ExtensionContext) {
       return
     }
 
-    if (!anyFolderNeedsLanguageServer(Workspace.workspaceFolders ?? [])) {
+    if (!await anyFolderNeedsLanguageServer(Workspace.workspaceFolders ?? [])) {
```

Comment:
🙈

Reply:
I know 🤦‍♂️

Reply:
I need to rework the typescript configs across the whole project to catch these things, turn on strict mode, etc…